examples/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ project(hipsycl-examples)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/../cmake)
 
-find_package(hipSYCL CONFIG REQUIRED)
+find_package(hipSYCL CONFIG PATHS /opt/hipSYCL/lib/cmake/ REQUIRED)
```

Comment:
Do we need to make the path user configurable if the user has installed hipSYCL in a custom location that is not found by cmake? I would propose to only make the hipSYCL installation prefix configurable, i.e.
```
find_package(hipSYCL CONFIG PATHS ${HIPSYCL_INSTALL_DIR}/lib/cmake/ REQUIRED)
```

Reply:
In the section of the search procedure of the CMake find_pacakge documentation, it says: 

`9. Search paths specified by the PATHS option. These are typically hard-coded guesses.`

If I understand correctly, the steps that come before 9 take precedence over `PATHS`, but that means the user should pass the `-DhipSYCL_DIR` flag which should point to the `HIPSYCL_INSTALL_DIR/lib/cmake` directory. if he or she wants to use hipSYCL installed to a custom location

I think we should check if the user set the `HIPSYCL_INSTALL_DIR` and then set the `hipSYCL_DIR`, before looking for the hipSYCL package.

Reply:
Ah, I forgot about `-DhipSYCL_DIR`. It's fine then. I think for the unit tests it's okay if the user has to remember to point it to `..../lib/cmake` since it will mostly be used by developers anyway.